features-json/colr.json

```diff
@@ -434,7 +434,7 @@
   "notes_by_num":{
     "1":"Supported only on Safari for macOS High Sierra (10.13)+",
     "2":"Supported only on Windows 8.1+",
-    "3":"On Windows pre 8.1 and macOS Sierra (10.12) and below Chrome [falls back to using FreeType](https://bugs.chromium.org/p/chromium/issues/detail?id=882844#c3)"
+    "3":"On Windows pre 8.1 and macOS Sierra (10.12) and below Chromium [falls back to using FreeType](https://bugs.chromium.org/p/chromium/issues/detail?id=882844#c3)"
```

Comment:
IMO, you can directly remove  `below Chromium` here, and the link is changed to: 

> falls back to using FreeType: [Chromium bug 882844](https://bugs.chromium.org/p/chromium/issues/detail?id=882844#c3)

Others, LGTM, thank you for your clarification.

Reply:
Did exactly that, thanks!

Reply:
Nice!